src/components/principal/unit/resource/StatusSelectionMenu.vue

```diff
@@ -80,6 +81,19 @@ export default {
       crew: []
     };
   },
+  computed: {
+    isSetStatusToMissionDisabled() {
+      return this.key === "" || this.description === "" || this.crew === "";
+    },
+    buildFormattedCrewList() {
+      let formattedCrew = this.resource.captain;
+      for (var i = 0; i < this.resource.crew.length; i++) {
```

Comment:
I'm not 100% sure but I believe  this.resource.crew.join('\n') would do the same.
join functions is used to transform an array into a string using a separator.